app/Resources/views/campaign/ton_macron.html.twig

```diff
@@ -0,0 +1,15 @@
+{{ introduction.content|replace({'%friend_first_name%': invitation.friendFirstName|e('html')})|raw }}
+
+<ul>
+{% for argument in invitation.arguments %}
+    {{- argument|raw }}
+{% endfor %}
+</ul>
+
+{{ conclusion.content|raw }}
+
+<p>
+{{ invitation.authorFirstName|e('html') }}
```

Comment:
Yes because author first name is filled by the user in the form. It's an input data.